src/compiler/crystal/tools/formatter.cr

```diff
@@ -1591,6 +1594,31 @@ module Crystal
       to_skip
     end
 
+    private def format_parameter_annotations(node)
+      if anns = node.parsed_annotations
```

Comment:
I'd suggest using early-return here to avoid indenting the whole method body:

```suggestion
      anns = node.parsed_annotations
      return unless anns
```